express/utils.py

```diff
@@ -28,3 +30,13 @@ def is_module_available(module_name: Union[str, Sequence[str]]) -> List[str]:
             module_version = importlib.metadata.version(module)
             logger.info(f"{module} version {module_version} available.")
     return missing_modules
+
+
+def get_cuda_availability():
+    """Function that checks for cuda device availability"""
+    try:
+        output = subprocess.check_output("nvidia-smi")  # nosec
+        logger.info("Nvidia GPU detected!")
+        logger.info(output.decode("utf-8"))
+    except FileNotFoundError:
```

Comment:
Is this the only error that can be raised?

Reply:
The two cases that I tested for was if we have the 

- nvidia drivers installed + no GPU
- nvidia drivers not installed + no GPU

and they both give the same error. I think those might be the only cases we have.